README.md

```diff
@@ -504,6 +504,21 @@ A literal object with:
     }
     ```
     will return only row for which row.column equal 'value'
+    the column name can be appended with the following modifier:
+        - not_: !=
+        - from_: cast to date and compare with >=
+        - to_: cast to date and compare with <=
+        - like_: ILIKE (match value with case insensitive)
```

Comment:
I think the `IS NULL`, etc. possibilities are not documented either. Might be worth doing it now :)